src/database/config.js

```diff
@@ -1,13 +1,14 @@
 require('dotenv').config();
 
+const username = process.env.DBUSERNAME || process.env.USERNAME;
 module.exports = {
   development: {
-    username: process.env.USERNAME,
+    username,
```

Comment:
why did you change line 6?

Reply:
was a workaround for some issues I had with my machine,  I'll undo it